src/Umbraco.Web.UI.Client/src/views/propertyeditors/slider/slider.controller.js

```diff
@@ -41,11 +41,11 @@
         }
         // convert to array
         $scope.sliderValue = $scope.model.value ? $scope.model.value.split(',') : null;
-        
+
         // don't render values with decimal places if the step increment in a whole number
         var stepDecimalPlaces = $scope.model.config.step % 1 == 0
             ? 0
-            : _.last($scope.model.config.step.toString().replace(",", ".").split(".")).length;
+            : $scope.model.config.step.toString().replace(",", ".").split(".")[$scope.model.config.step.toString().replace(",", ".").split(".") - 1].length;
```

Comment:
You should probably only do `$scope.model.config.step.toString().replace(",", ".").split(".")` once and store the value so you don't calculate it twice. 

Or if you want to get fancy, you could do `$scope.model.config.step.toString().replace(",", ".").split(".").slice(-1)[0]`

Reply:
Yeh I was trying to not change too much and be quite literal with my first pass/conversions, but good idea :)